src/sentry/relay/projectconfig_cache/redis.py

```diff
@@ -44,11 +31,14 @@ def set_many(self, configs):
         # Note: Those are multiple pipelines, one per cluster node
         p = self.cluster.pipeline()
         for public_key, config in configs.items():
-            value = json.dumps(config)
-            if _use_compression(public_key):
-                value = zstandard.compress(value.encode(), level=COMPRESSION_LEVEL)
-            metrics.timing("relay.projectconfig_cache.size", len(value))
-            p.setex(self.__get_redis_key(public_key), REDIS_CACHE_TIMEOUT, value)
+            serialized = json.dumps(config).encode()
+            compressed = zstandard.compress(serialized, level=COMPRESSION_LEVEL)
+            compressed_size = len(compressed)
+            compression_ratio = len(serialized) / compressed_size
+            metrics.timing("relay.projectconfig_cache.size", compressed_size)
```

Comment:
please also add uncompressed size

Reply:
Just refactored to do so. I think we can omit the actual compression ratio, it might not be recoverable in a mathematically correct way from the statsd metric, but the two sizes give us a rough idea.